packages/foam-vscode/src/utils.ts

```diff
@@ -91,3 +91,35 @@ export function dropExtension(path: string): string {
 export const astPositionToVsCodePosition = (point: Point): Position => {
   return new Position(point.line - 1, point.column - 1);
 };
+
+/**
+ * Used for the "Copy to Clipboard Without Brackets" command
+ *
+ */
+export function removeBrackets(s: string): string {
+  // take in the string, split on space
+  const stringSplitBySpace = s.split(" ");
+
+  // loop through words
+  const modifiedWords = stringSplitBySpace.map(currentWord => {
+    if (currentWord.includes("[[")) {
+      let word = currentWord.replace(/(\[\[)/g, "");
+      word = word.replace(/(\]\])/g, "");
+      word = word.replace(/(.mdx|.md|.markdown)/g, "");
```

Comment:
Do we have this list somewhere already? Should we consolidate it or do you think it's better to keep things more decoupled at the moment?

Reply:
Good call! I didn't even think to check. I'll look and if it is, I'll use that

Reply:
Okay I found it inside `docs/_layouts/foam.html`, but can't export it because it's just in a script tag in an HTML file 🤔

I'm not seeing it elsewhere...I'm going to leave for now, but if you see a better option, let me know and I'll happily change it!